graphics/opengl/debug.h

```diff
@@ -24,7 +24,9 @@
 
 #include "graphics/opengl/context.h"
 
+#ifndef __EMSCRIPTEN__
```

Comment:
maybe this should get wrapped in in `#ifdef DEBUG` - other platforms might have the same problem.

Reply:
@ccawley2011 can you chime in? Considering that nobody has noticed this so far since 18 months ( scummvm/scummvm@863988f), I want to be careful to disable it for all platforms, even if it's only in non-debug builds.

Reply:
If it's just slow because there are a lot of warnings, I'd leave this as specific to Emscripten for now.